dotnet/src/Extensions/Extensions.UnitTests/TemplateEngine/PromptTemplateEngineTests.cs

```diff
@@ -9,11 +9,11 @@
 using Microsoft.SemanticKernel.TemplateEngine;
 using Microsoft.SemanticKernel.TemplateEngine.Blocks;
 using Moq;
-using SemanticKernel.UnitTests.XunitHelpers;
+using SemanticKernel.Extensions.UnitTests.XunitHelpers;
 using Xunit;
 using Xunit.Abstractions;
 
-namespace SemanticKernel.UnitTests.TemplateEngine;
+namespace SemanticKernel.Extensions.UnitTests.TemplateEngine;
```

Comment:
Should we add some tests for the new interface method, too?

Reply:
I have removed the method